addons/sale/controllers/portal.py

```diff
@@ -155,7 +155,7 @@ def portal_order_page(self, order_id, report_type=None, access_token=None, messa
 
         # Log only once a day
         if order_sudo and request.session.get('view_quote_%s' % order_sudo.id) != now and request.env.user.share and access_token:
```

Comment:
```py
request.session.get('view_quote_%s' % order_sudo.id) != now
```

need to be updated to take the change in format into account. Note that you can't assume that it's isoformat, since the session might contain data from before your patch.